assets/javascripts/modules/analytics/ga.js

```diff
@@ -30,12 +36,14 @@ define(['utils/cookie',
          */
         ga('require', 'linkid', 'linkid.js');
 
-        ga('set', 'dimension1', identitySignedIn.toString());   // deprecated
```

Comment:
Did you want to do this immediately in this first iteration?
Or can we leave it in for the time being to check against new data. Turning it off from now might affect a dashboard that is in use.